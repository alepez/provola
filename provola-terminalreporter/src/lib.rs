mod simple;

pub use simple::TerminalReporter;
