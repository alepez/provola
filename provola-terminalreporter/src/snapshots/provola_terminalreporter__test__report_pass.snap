---
source: src/lib.rs
expression: s

---
"PASS\n"
